### PR TITLE
Do not raise exceptions related to empty databases

### PR DESCRIPTION
**Pull request by morenol**

### CODE REVIEW

File: eox_core/api/v1/permissions.py [hunk 1]
```diff
@@ -15,12 +16,19 @@ def load_permissions():
...
-        )
+        try:
+            content_type = ContentType.objects.get_for_model(User)
+            obj, created = Permission.objects.get_or_create(  # pylint: disable=unused-variable
+                codename='can_call_eox_core',
+                name='Can access eox-core API',
+                content_type=content_type,
+            )
+        except ProgrammingError as err:
+            # Do not raise exception if it is related to database migrations
```
Comment (jfavellar90) [inline eox_core/api/v1/permissions.py]: I would add just a more detailed explanation about the issue, indicating that this code runs loading the app before running the migrations, and for that reason we're bypassing the ProgrammingError in this case, to let migrations run smoothly.

File: eox_core/api/v1/permissions.py [hunk 2]
```diff
@@ -15,12 +16,19 @@ def load_permissions():
...
+            pass
+        except Exception as err: # pylint: disable=broad-except
```
Comment (felipemontoya) [inline eox_core/api/v1/permissions.py]: Do we need this, we can just not catch the other exceptions

File: eox_core/api/v1/permissions.py [hunk 3]
```diff
@@ -15,12 +16,17 @@ def load_permissions():
...
-            content_type=content_type,
-        )
+        try:
+            content_type = ContentType.objects.get_for_model(User)
+            obj, created = Permission.objects.get_or_create(  # pylint: disable=unused-variable
+                codename='can_call_eox_core',
+                name='Can access eox-core API',
+                content_type=content_type,
+            )
+        except ProgrammingError as err:
```
Comment (jfavellar90) [inline eox_core/api/v1/permissions.py]: just a fix the pylint problem here (err defined but not used)

**Comment by diegomillan** [comment]
Can we rebase and merge @morenol ??

**Comment by morenol** [comment]
@diegomillan rebased

**Comment by diegomillan** [comment]
I will merge @morenol

**Pull request merged**